components/Details.tsx

```diff
@@ -39,6 +39,7 @@ export const Details: React.FunctionComponent<Props> = (props) => {
         }
 
         summary {
+          outline: none;
```

Comment:
What motivated this change?

Reply:
It was a request made by Gill on the ticket to remove the outline around the boxes. Does it need to be there due to accessibility standards?

Reply:
Yeah. It's an accessibility need to highlight where the current focus is, which is what the outline is for.